kratos/modeler/connectivity_preserve_modeler.h

```diff
@@ -131,6 +135,22 @@ class KRATOS_API(KRATOS_CORE) ConnectivityPreserveModeler : public Modeler
         const Condition& rReferenceCondition
     );
 
+    /// Generate a copy of rOriginModelPart in rDestinationModelPart.
+    /** This function fills rDestinationModelPart using data obtained from
```

Comment:
```suggestion
    /**
     * @briefGenerate a copy of rOriginModelPart in rDestinationModelPart.
     * @details This function fills rDestinationModelPart using data obtained from
```

Reply:
you're missing a space between *brief* and *Generate*

Reply:
Done. Thanks for noting it.